src/components/calcite-action-group/calcite-action-group.scss

```diff
@@ -1,3 +1,11 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --columns: sets number of grid-template-columns when layout is grid
```

Comment:
Should this be `--calcite-action-group-columns`?

Reply:
i'll change 👍